.github/workflows/approve-lake-gate.yml

```
@@ -25,6 +25,40 @@ jobs:
         fetch-depth: 0
         token: ${{ secrets.GITHUB_TOKEN }}
 
+    - name: Guard: disallow forks
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 28: mapping values are not allowed in this context

(syntax-check)